docs/external_access.md

```diff
@@ -0,0 +1,89 @@
+
+Apply the manifests.
+
+```sh
+kubectl apply -f config/samples/external_access/cert-manager-issuer.yaml
+kubectl apply -f config/samples/external_access/cert-manager-certificate.yaml
+```
+
+### Create the MongoDB deployment
+
+Edit ```config/samples/external_access/mongodb.com_v1_mongodbcommunity_cr.yaml```. Replace <mongodb-name> with the desired MongoDB deployment name -- this should be the same as in the previous step. Replace ```<domain-rs-1>```, ```<domain-rs-2>```, and ```<domain-rs-3>``` with the external FQDNs of the MongoDB replicaset members. Please remember that you should have the same number of entries in this section as the number of your replicaset members. You can also edit the ports for external access to your preferred numbers in this section -- you will remmeber to change them in the next step too. Change ```<your-admin-password>``` to your desired admin password for MongoDB.
+
+Apply the manifest.
+
+```sh
+kubectl apply -f config/samples/external_access/mongodb.com_v1_mongodbcommunity_cr.yaml
+```
+
+Wait for the replicaset to be available.
+
+### Create the external NodePort services for accessing the MongoDB deployment from outside the Kubernetes cluster
+
+Edit ```config/samples/external_access/external_services.yaml``` and replace ```<mongodb-name>``` with the MongoDB deployment name that you have used in the preceeding steps. You can change the ```nodePort``` and ```port``` to reflect the changes (if any) you have made in the previous steps.
+
+Apply the manifest.
+
+```sh
+kubectl apply -f config/samples/external_access/external_services.yaml
+```
+
+### Retrieve the certificates from a MongoDB replicaset member
+
+```sh
+kubectl exec --namespace mcommunity -it <mongodb-name>-0 -c mongod -- bash
+```
+
+Once inside the container ```cat``` and copy the contents of ```/var/lib/tls/ca/ca.crt``` into a file on your local system. Also copy the contents of the ```.pem``` file in ```/var/lib/tls/server``` into a file on your local system.
+
+### Connect to the MongoDB deployment from outside the Kubernetes cluster
+
+This is an example to connect to the MongoDB cluster with Mongo shell. Use the certificates from the previous step. Replace the values in the command from the preceeding steps.
+
+```sh
+mongosh --tls --tlsCAfile ca.crt --tlsCertificateKeyFile key.pem --username my-user --password <your-admin-password> mongodb://<domain-rs-1>:31181,<domain-rs-2>:31182,<domain-rs-3>:31183
+```
+
+### Conclusion
+At this point, you should be able to connect to the MongoDB deployment from outside the cluster. Make sure that you can resolve to the FQDNs for the replicaset members where you have the Mongo client installed.
```

Comment:
nit : maybe add an example here as to how to check if FQDN is resolvable using `dig` or something.

Reply:
Seems OOS to me.